server/bin/www

```diff
@@ -31,11 +34,47 @@ const server = http.createServer(app);
 
 const io = require("socket.io")(server);
 
+// on a new connection
```

Comment:
Be mindful of not overusing comments -a lot of times, the code can speak for itself